docs/tutorials/introduction.ipynb

```diff
@@ -0,0 +1,653 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "-"
+    }
+   },
+   "source": [
+    "# Introduction\n",
+    "In this tutorial, we will see how one can use `audtorch` to rapidly speed up the development of audio-based deep learning applications."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "-"
+    }
+   },
+   "source": [
+    "## Preliminaries\n",
+    "\n",
+    "* [PyTorch](https://pytorch.org/) already has an inteface for data sets, aptly called [Dataset](https://pytorch.org/docs/stable/data.html#torch.utils.data.Dataset) \n",
+    "* It then wraps this interface with a [DataLoader](https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader) that efficiently allows us to loop through the data in parallel, and takes care of the random order as well\n",
+    "* All we need to do is implement the [Dataset](https://pytorch.org/docs/stable/data.html#torch.utils.data.Dataset) interface to get the input for the model and the labels\n",
+    "* **However**, it is not easy for beginners to see how one can go from a bunch of files in their hard drive, to the features that will be used as input in a machine learning model\n",
+    "* **Thankfully**, `audtorch` is there to take of all that for you :-)\n",
+    "\n",
+    "Before you start, you might want to familiarize yourselves with [PyTorch's data pipeline] (https://pytorch.org/docs/stable/data.html)"
```

Comment:
There is a typo, should be `[PyTorch's data pipeline](https://pytorch.org/docs/stable/data.html)`